### PR TITLE
NO-ISSUE: Add event name documentation

### PR DESCRIPTION
**Pull request by arielireni**
Added documentation regarding events name format.

Signed-off-by: arielireni <aireni@redhat.com>

# Assisted Pull Request

## List all the issues related to this PR

- [ ] New Feature
- [ ] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [ ] Cloud
- [ ] Operator Managed Deployments
- [x] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [ ] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [x] No tests needed

## Assignees

/cc @
/cc @

## Checklist

- [ ] Title and description added to both, commit and PR.
- [ ] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [ ] Reviewers have been listed
- [ ] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- [ ] Are the title and description (in both PR and commit) meaningful and clear?
- [ ] Is there a bug required (and linked) for this change?
- [ ] Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

**Comment by arielireni** [comment]
\assign @masayag @nmagnezi

File: docs/dev/events.md [hunk 1]
```diff
@@ -16,7 +16,8 @@ In order to add a new event, follow the next steps:
 ### Event Definition
 Event definition should specify the following attributes:
 1. __name__: A unique name of the event. The name needs to remain unique and constant
-as it may be referred by the service's clients (e.g. by the UI).
+as it may be referred by the service's clients (e.g. by the UI). The name should match
+the structure <event-context>_<past_tense>.
```
Comment (masayag) [inline docs/dev/events.md:20]: This doesn't look good when rendered

Selection_999(461)

Please add escaping or code format tags around `<event-context>_<past_tense>`

Comment (arielireni) [reply docs/dev/events.md]: done

**Review by masayag — APPROVED** [review]

**Review by nmagnezi — APPROVED** [review]
/lgtm
/approve

**Pull request merged**